example-scripts/01-sample-script.py

```diff
@@ -0,0 +1,57 @@
+"""Module 1 - 
+
+* What challenges do you see in this code? 
```

Comment:
This would be an OYO (on your own) or in small groups discussion that learners would have around the code.